src/index.js

```diff
@@ -8,6 +8,7 @@ import { Provider } from 'react-redux';
 import App from './components/App';
 import INITIAL_STATE from './constant/initialState';
 import rootReducer from './reducers';
+import './styles/index.scss';
 
 const STATE = { books: INITIAL_STATE };
 
```

Comment:
Can you remove this code from your `createStore` parameters?
```js
    window.__REDUX_DEVTOOLS_EXTENSION__
                            && window.__REDUX_DEVTOOLS_EXTENSION__()));
```
This is resulting in an error shown below:
![image](https://user-images.githubusercontent.com/36057474/90309344-c3118500-dedf-11ea-9007-1a39abe24431.png)


This is the same error reported by @fatymahmed earlier which means it's not from my side alone.


I can see it also affects [your live link ](https://shrouded-brushlands-72459.herokuapp.com/) in your Readme.

![image](https://user-images.githubusercontent.com/36057474/90309394-256a8580-dee0-11ea-9f06-b91fb5a2112d.png)